README.md

```diff
@@ -60,6 +61,35 @@ which contain a toctree need to have the suffix `.trst` instead of just `.rst`.
 This way the projects which use this as a submodule are able to ignore the
 toctrees and build their own instead.
 
+### Understanding Spell Check
+A spell check is performed during the build step by default. It uses in-built
+language specific dictionaries and project specific dictionaries (aspell.en.pws)
+to verify the spellings and causes the build to fail in case of any typos.
+
+The project, which uses this blueprint, should have its own custom dictionary,
+similar to the one in blueprint (aspell.en.pws). A link to the check_spelling.sh
```

Comment:
I'm guessing that the location of the project specific dictionary is important in order for the `check_spelling.sh` script to find it? Can you mention where to put the dict for project specific checks?

Reply:
Also, I believe the location of the symlink is also important (providing that you try finding the project specific dict based on the location of the symlink) - can you mention where the symlink should be created in the project specific case?